eslint.config.mjs

```diff
@@ -37,6 +38,13 @@ const defaultEslintConfig = tseslint.config(
 
 export default [
   ...defaultEslintConfig,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.node,
+      },
+    },
+  },
```

Comment:
Uten dette får jeg feilmeldinger om at `'require' is not defined` og `'console' is not defined` i CustomPostCSSLoader.js.